src/modules/partition/core/PartitionLayout.cpp

```diff
@@ -28,17 +31,32 @@
 #include <kpmcore/core/partition.h>
 #include <kpmcore/fs/filesystem.h>
 
+static int
```

Comment:
That looks poorly-typed to me: you're going from FileSystem::Type to int. It should be the enum type, and clients somewhere else can cast it to int if needed (besides, this will break if FileSystem::Type changes to an enum class).

Reply:
I'm pretty sure I had a good reason to use int back then. As I can't even remotely remember why, I guess it wasn't so good, after all...